docs/advanced-features/dynamic-import.md

```diff
@@ -11,7 +11,7 @@ description: Dynamically import JavaScript modules and React Components and spli
   </ul>
 </details>
 
-Next.js supports ES2020 [dynamic `import()`](https://github.com/tc39/proposal-dynamic-import) for JavaScript. With it you can import JavaScript modules (inc. React Components) dynamically and work with them. They also work with SSR.
+Next.js supports ES2020 [dynamic `import()`](https://github.com/tc39/proposal-dynamic-import) for JavaScript. With it you can import React Components dynamically and work with them. They also work with SSR.
```

Comment:
`import()` supports anything so I guess that you're just confused on that `dynamic()` only works for React components 🤔 Maybe that should be explained more

Reply:
Yeah exactly @timneutkens, I was trying to clarify here that `dynamic()` is only for components.

> They also work with SSR.

This part, is only true for `dynamic()` imports right? If so the text might need rephrasing in general.

Reply:
> This part, is only true for dynamic() imports right? If so the text might need rephrasing in general.

`import()` works fine server-side as well, it's only useful in a few cases though (e.g. using them in getServerSideProps/getStaticProps)